libs/components/src/popover/vl-popover.linked-list.css.ts

```diff
@@ -0,0 +1,43 @@
+import { css, CSSResult } from 'lit';
+
+const styles: CSSResult = css`
+    ::slotted(.vl-link-list) {
+        padding: 0;
+
+        li,
```

Comment:
Werken deze styles? Voor zover ik weet kan je enkel de buitenste tag van een slotted element stylen op deze manier. Ik weet ook niet of het goed is om deze styles hier toe te voegen, van zodra we van de linked-list een Lit component met shadow DOM maken gaat dit niet meer werken.

Reply:
ja dat werkt; kan dit eventueel eens demonstreren >

background toevoegen onder die padding doet dit:

<img width="252" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/c7fb0a73-3344-45ba-81ce-d2eeb74c63ca">

---

maar ik ga akkoord, dat dit naar toekomst toe dan niet meer op dezelfde manier zal werken; heb hier vooral voorbeeld van DV gevolgd die (zoals bij vele componenten) in hun parent component styling aanbieden voor de verwachte child componenten (en dit dus ook doen voor hun `vl-popover` wanneer die een `vl-ui-link-list` heeft)

maar dat is mss niet per sé een voorbeeld die we willen blijven volgen gezien de extra complexiteit met shadow dom & zoals je zelf aangeeft, de toekomstige uitdaging wanneer we afstappen van `is="vl-link-list"`

voor mij goed om dit dan te verwijderen

@kspeltix wat denk jij?

Reply:
zoals besproken, voegen we dus `vl-popover-action` & `vl-popover-action-list` toe